src/views/HousingLottery/HousingLottery.module.scss

```diff
@@ -1,25 +1,52 @@
-.submit_button {
-  position: "absolute";
-  margin: "auto";
-  top: 15px;
-  bottom: 0;
-  left: 45em;
-  right: 20em;
-  height: 3em;
-  width: 9em;
+.submit_preference_button {
+  height: 45px;
+  margin-left: 10px;
 }
 
 .add_applicant_button {
-  position: "absolute";
-  margin: "auto";
-  top: 15px;
-  bottom: 0;
-  left: 43em;
-  right: 5em;
-  height: 3em;
-  width: 11em;
+  height: 45px;
+  margin-left: 10px;
+}
```

Comment:
For similar reason as `.submit_preference_button`, I will be deleting the `.add_applicant_button` because we are no longer in need of the button. However, I will reference the Button components that we already have to fix other issues.